app/views/content/blog/why-i-became-a-teacher.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Why I became a teacher
+date: "2021-02-01"
+images:
+  giorgio_rubbo:
+    path: "media/images/content/blog/giorgio-rubbo.jpg"
+    alt: "A photograph of a teacher chatting with their student in a classroom" 
+description: |-
+  description
```

Comment:
Perhaps something like:
Giorgio Rubbo talks about what inspired him to become a teacher, his application process, concerns he had, and why you should become a teacher if you're thinking about it.